packages/SwingSet/src/controller/controller.js

```diff
@@ -75,7 +75,7 @@ function makeConsole(tagOrTagCreator) {
 function unhandledRejectionHandler(e, pr) {
   // Don't trigger sensitive hosts (like AVA).
   pr.catch(() => {});
-  console.error('UnhandledPromiseRejectionWarning:', e);
+  console.error('🚨 UnhandledPromiseRejection:', e);
```

Comment:
or maybe 📣  since it's not necessarily a problem. 

alternately we can treat all failures as a problem and have an explicit catch for "expected to never happen" or "ignore failure"

Reply:
# 🤞